src/RobBrazier/Piwik/Module/UsersManagerModule.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace RobBrazier\Piwik\Module;
+
+/**
+ * Class SitesManagerModule.
+ *
+ * @see https://developer.matomo.org/api-reference/reporting-api#SitesManager for arguments
```

Comment:
Can this link be updated to "https://developer.matomo.org/api-reference/reporting-api#UsersManager"